public/css/syntax.css

```diff
@@ -28,14 +28,14 @@
 .s { color: #ce3512 } /* Literal.String */
 .na { color: #006699; } /* Name.Attribute */
 .nb { color: #336666 } /* Name.Builtin */
-.nc { color: #00AA88; } /* Name.Class */
+.nc { color: #007961; } /* Name.Class */
```

Comment:
question: This syntax.css could it have come from some library somewhere and as such we could look at whether they have updates... :-)